src/site/markdown/server/manage-webadmin.md

```diff
@@ -3330,6 +3214,37 @@ Response codes:
 
  - 200: A list of corresponding tasks is returned
  - 400: Invalid status value
+ 
+### Endpoints returning a task
+
+Many endpoints do generate a task.
+
+Example:
+
+```
+curl -XPOST /endpoint?action=XXX
+```
+
+The response to these requests will be the scheduled `taskId` :
+
+```
+{"taskId":"5641376-02ed-47bd-bcc7-76ff6262d92a"}
+```
+
+Positionned headers:
+
+ - Location header indicates the location of the resource associated with the scheduled task. Example:
+
+```
+Location: /tasks/3294a976-ce63-491e-bd52-1b6f465ed7a2
+```
+
+Response codes:
+
+ - 201: Success. Corresponding task id is returned.
+ - Other response codes might be returned depending on the endpoint
+
+The additional information of such tasks depends on the scheduled task and is documented in the endpoint documentation.
```

Comment:
The additional information returned depends on the scheduled task type and is documented in the endpoint documentation.
WDYT ?